python_service/main_grpc_server.py

```
@@ -0,0 +1,37 @@
+    Implements the gRPC service definition for file sanitization.
+    """
+    def SanitizeDocument(self, request, context):
+        """
+        Processes a gRPC request to sanitize a document.
+        """
+        try:
+            sanitized = sanitize_file(request.document_data, request.document_type)
+            return sanitizer_pb2.SanitizeResponse(sanitized_content=sanitized)
+        except ValueError as e:
+            context.abort(grpc.StatusCode.INVALID_ARGUMENT, str(e))
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)